### ISSUE TITLE
 DetailsList column headers not accessible when selectionMode=SelectionMode.single using Keyboard navigation

### ISSUE THREAD

**Issue by arunarokk** [OfficeDev/office-ui-fabric-react#4813]
### Bug Report

- __Package version(s)__: (5.96.0)
- __Browser and OS versions__: (browser- chrome, edge and windows 10)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Normal) 

Products/sites affected: (if applicable)

#### Describe the issue:

when selectionmode is single for the detailslist, keyboard navigation to column headers is not working, with tab its directly moving row instead of column header. with selection mode multiple its working fine.

**Comment by natalieethell** [comment]
Hi @arunarokk thanks for submitting this issue! Here's a screen capture for visibility:

![detailslist can t focus on columns](https://user-images.githubusercontent.com/14134000/41185210-f45dc2ce-6b3a-11e8-80a0-0fb144e11913.gif)

It looks like we have two options for a fix. In single selectionmode, either we can make the "check all" checkbox focusable, but not selectable or we can just not make the "check all" checkbox focusable at all. I, personally, am leaning towards the second option, as it doesn't really make sense to make the "check all" checkbox focusable if you aren't using it for anything.

**Comment by natalieethell** [comment]
Spoke offline. We will make the "check all" checkbox non-focusable in single selectionmode.
